### ISSUE TITLE
Extra space in expressions of sequences

### ISSUE THREAD

**Issue by mahrud** [Macaulay2/M2#3502]
```m2
i1 : R = kk[x,y];

i2 : net ideal x

o2 = ideal x

i3 : net ideal(x+y)

o3 = ideal(x + y)
```
Now notice the extra space before the parentheses:
```
i4 : net ideal(x+y,x-y)

o4 = ideal (x + y, x - y)

i5 : net ideal(x,y)

o5 = ideal (x, y)
```
@pzinn do you have a guess of what's causing this?

**Comment by pzinn** [comment]
Yeah it's been like this for as long as I can remember. During my `expression` upgrade I don't think I changed this behaviour. It's controlled by `net Adjacent` in `expressions.m2`. You can try to fiddle with it, but beware of unintended consequences. Also, you can compare with `texMath Adjacent` (which I suspect I wrote), which has slightly different spacing behaviour. (in particular the issue you mention doesn't arise in WebApp mode).

**Comment by d-torrance** [comment]
> **Written by Claude** (Claude Opus 5, via Claude Code), not by @d-torrance, whose account posted it -- please weigh it accordingly.

Reproduces on 1.26.06 — the transcript needs `kk` replaced by a real ring to run at all. Completing
the `texMath Adjacent` comparison @pzinn suggested, because it turns out to contain the fix.

`net Adjacent` (`m2/expressions.m2:660`) branches on `precedence args > p`. A `Sequence` argument
scores 74 against the `Adjacent`'s 62, so it takes `:669` — and the exemption list at `:671`
(`Array`, a `Divide`, or a multi-line `netfun`) does not cover `Sequence`, so it falls through to
`:673`:

```m2
else horizontalJoin (netfun, " ", netargs)
```

A `Sum` argument scores 50, below 62, so it takes `:677` instead and parenthesizes itself with no
separator. That asymmetry is the whole of it. And `texMath Adjacent` at `:688-690` already handles
the identical case — `instance(args,VisibleList)` gets `sep = "{}"`, zero width — which is why the
WebApp output has no gap. Adding `VisibleList` to the `:671` exemption, matching `Array`, is the same
fix one branch over.

On "beware of unintended consequences", with that change patched in and measured:

- Ten of twelve sampled renderings change, each **exactly one column narrower**, with height, depth
  and multi-line alignment preserved — including `ideal (x^7 + y^7, ...)` with its stacked exponents
  and `ideal (x/y, y/z)`, whose fraction bars shift together. Single-generator ideals and bare
  cokernels are untouched.
- **Brace renderings are untouched**, which was the risk, since `VisibleList` covers `List` as well
  as `Sequence`: `set {…}` passes its argument as a `Holder`, and `cokernel` and `image` pass a
  `MatrixExpression`, neither of which is a `VisibleList`. `Array` arguments already have no space.
- No test anywhere asserts on the spacing.
- The committed diff is **36 output lines in 14 files** — the pre-computed `.out` files for packages
  that need external software. Around 3800 lines across 1296 `example-output` files would also
  change, but those are build artifacts: `git ls-files` finds no `example-output` path.

<sub>Triaged in a sweep of the open issues; source of truth: [`bug-triage/issues.tsv`](https://github.com/d-torrance/M2/blob/bug-triage/bug-triage/issues.tsv).</sub>
